### ISSUE TITLE
Large mobile device on-screen keyboard opens when hovering or performing a switch or move

### ISSUE THREAD

**Issue by MarTnquesada** [smogon/pokemon-showdown-client#1925]
Mobile on-screen keyboard opens when hovering or performing a switch or move. This stops happening when the chat is not focused/selected, either because the device is vertical (like a phone) or turning the tablet vertically so that chat is hidden, or by clicking on a section of the background that appears at the top of the screen. But when using Showdown with the battle box and chat open at the same time, hovering over a pokemon or clicking on the main box in any way focuses the chat textbox (the cursor appears in the textbox) and therefore the on-screen keyboard opens whenever you hover or perform a switch or move.

Thought that it could be because of using an IME but getting rid of all languages save for the basic US English Qwerty keyboard does not seem to make a difference.

Replicated on Samsung Galaxy Tab S7 FE with Android 11 (uses OneUI 3.1 since its a Samsung tablet) for latest releases of Chrome, Opera and Firefox.

**Comment by Zarel** [comment]
What does "hovering" mean in a mobile context? Long-pressing?

**Comment by MarTnquesada** [comment]
Yes, in this context I was referring to long-pressing, i.e. when you want to hover to check one of your mons.

**Comment by joebangara43903** [comment]
![image](https://user-images.githubusercontent.com/131228993/232965421-6892b357-e34f-4f02-81a8-6cd67d7508a7.png)
